### PR TITLE
Fixes #16778 - Added check for /dev/log on installer

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

File: hooks/pre/19-devlog.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# check to see if /dev/log directory exists
+
+dir_present = File.exist?('/dev/log')
```
Comment (ekohl) [inline hooks/pre/19-devlog.rb]: Maybe move this to the elsif statement so skipping the check really does skip everything related?

**Comment by chris1984** [comment]
@ekohl fixed

File: hooks/pre/19-devlog.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# check to see if /dev/log directory exists
+
+if app_value(:disable_system_checks)
+  logger.warn 'Skipping system checks.'
+  # if dir_present is true then give warning to user and exit otherwise move on withe install
```
Comment (sthirugn) [inline hooks/pre/19-devlog.rb]: s/withe/with the

File: hooks/pre/19-devlog.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+elsif
```
Comment (ekohl) [inline hooks/pre/19-devlog.rb]: Shouldn't this be on a single line?

**Comment by chris1984** [comment]
@ekohl @sthirugn fixed the typo and added the rubocop fix with the true/false statement on the same line as the elsif.

**Comment by jlsherrill** [comment]
Thanks @chris1984, merging!

**Pull request merged**